README.md

```diff
@@ -140,15 +183,47 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+A RESTful route tells the controller how to handle the Emperor's demands, and
+what information it needs to retrieve from the database to render on the page.
 ```
 Your routes:
 ```
 The ancestors have provided an example of one route; you do the other six!
 
+GET '/warriors'
+  * This is the index route, which displays all warriors
+
+GET '/warriors/new'
+  * This is the new route, which displays a form where you can enter information
+  about a new warrior.
+
 GET '/warriors/:id'
   * This is the show route, which finds a warrior by ID, and displays information about that warrior.
 
+GET '/warriors/:id/edit'
+  * This is the edit route, which finds a warrior by ID and displays a page that
+  includes a delete button, should your warrior fall in battle, or just displeases you,
+  and a form where you can update a specific warrior's information.
```

Comment:
Those disappointing warriors, never quite giving that 110% that we have grown to expect.  At least they're easy to replace. <click>.